larpandoracontent/LArThreeDReco/LArHitCreation/HitCreationBaseTool.cc

```diff
@@ -78,6 +80,7 @@ void HitCreationBaseTool::GetBestPosition3D(const HitType hitType1, const HitTyp
     // TODO Input better uncertainties into this method (sigmaHit, sigmaFit, sigmaX)
     const CaloHit *const pCaloHit2D(protoHit.GetParentCaloHit2D());
     const HitType hitType(pCaloHit2D->GetHitType());
+    const LArTPCMap &larTPCMap(this->GetPandora().GetGeometry()->GetLArTPCMap());
```

Comment:
I think this could be initialised immediately prior to the loop in which it is used